### ISSUE TITLE
Account Creation Issues

### ISSUE THREAD

**Issue by user5423** [AR-Reshare/AR-Reshare#25]
Account Creation should be pulled out of the regular SecurityValidate class and moved over to a miscellaneous class (i.e. not part of the Pipeline) - this should be done so that we can better accommodate sending account creation emails to the user.

**Comment by zshepherd01** [comment]
This component is waiting on #33
